packages/flutter/lib/src/rendering/unconstrained_box.dart

```diff
@@ -0,0 +1,170 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+/// Renders an [UnconstrainedBox], imposing no constraints on its child,
```

Comment:
RenderObjects can be used in isolation (without widgets). They are an API layer of their own. so the description here should ideally not start by referencing the widgets, since not all consumers will be using the widgets.

(It's fine to reference the widgets later, as you do)